modules/cli-manager-adding-plugins.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/cli_manager/cli-manager-using.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cli-manager-adding-plugins_{context}"]
+= Adding CLI plugins with the CLI Manager
```

Comment:
Along the lines of my other comment about not using adding for both admins/users, maybe this should be installing? Because they're installing something that the admin already made available?